src/clevertap.js

```diff
@@ -572,7 +576,13 @@ export default class CleverTap {
       // already initailsed
       return
     }
-    StorageManager.removeCookie('WZRK_P', window.location.hostname)
+
+    if (!isWindowDefined() && typeof browser === 'object') {
+      this.#account.mode = 'SHOPIFY'
+      StorageManager.saveMode('async')
```

Comment:
thinking of using mode here rather than defining a new variable. Will update this if required.